src/sortStudents.ts

```diff
@@ -1,16 +1,92 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(students: Student[],
+  sortBy: SortType, order: SortOrder): Student[] {
```

Comment:
Please, fix it as I said in the previous review. It still hard to read 
```suggestion
export function sortStudents(
  students: Student[],
  sortBy: SortType, 
  order: SortOrder
): Student[] {
```